version/8_11_4.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 
-NODE_VERSION=8.11.3
+NODE_VERSION=8.11.4
 echo "=============== Installing Node $NODE_VERSION ============="
 . /root/.nvm/nvm.sh && nvm install $NODE_VERSION
-nvm use $NODE_VERSION && npm install npm@6.3.0 -g
-nvm alias default 8.11.3
+nvm use $NODE_VERSION && npm install npm@6.4.0 -g
```

Comment:
Please make this consistent with what we are doing in https://github.com/dry-dock/u16nodall/pull/39/files#diff-67a257f98a241301b4659a61724e5ee2